src/Cli/dotnet/CommonLocalizableStrings.resx

```diff
@@ -523,6 +523,9 @@
   <data name="ToolSettingsUnsupportedRunner" xml:space="preserve">
     <value>Command '{0}' uses unsupported runner '{1}'."</value>
   </data>
+  <data name="SolutionExecutableConfigurationMismatchError" xml:space="preserve">
+    <value>Multiple executable projects in the solution contain conflicting '{0}' values. See '{1}'. Make the values match. Consider using a Directory.build.props file to set all project configurations.</value>
```

Comment:
Since the list of projects is going to have newlines in it, I would move it to the end:
```suggestion
    <value>Multiple executable projects in the solution contain conflicting '{0}' values. Make the values match. Consider using a Directory.build.props file to set all project configurations.  Conflicting projects:
{1}</value>
```